gist-backup

```diff
@@ -97,10 +107,10 @@ retries=0
 MAX_RETRIES=5
 while [ $retries -lt $MAX_RETRIES ]
 do
-  echo "Requesting Gist page: $page"
+  echo "Requesting Gist page: $page" from $url
```

Comment:
There's a minor issue here with the `"`s around the full content, but I can fix that up on `master` after merging.